src/cpp_common/utilities.cpp

```
@@ -0,0 +1,129 @@
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+
+#include "cpp_common/utilities.hpp"
+
+#include <string>
+
+#include "c_common/enums.h"
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)